client/electron/package.js

```diff
@@ -138,9 +138,14 @@ const options = {
     guid: '2f56a772-db54-4a32-b264-28c42970f684',
   },
 
+  afterSign: 'electron-builder-notarize',
```

Comment:
Should be set to `undefined` if `OPTS.sign` is false (to allow to build an app that is not signed)